.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,51 +7,64 @@ assignees: ''
 
 ---
 
-**Describe the bug**
-<!--A clear and concise description of what the bug is.-->
+<!-- 
+Explanation of applied labels can be found here: https://github.com/kirbydesign/designsystem/labels
 
-**To Reproduce**
-Steps to reproduce the behavior:
+The good issue: contains only one issue, is self-contained, is specific & unambigious, follows the template, has a good title, is easy to read and contains screenshots & -recordings
+-->
+
+[ ] I have written a [good issue](../CONTRIBUTING.md/#issue) 
+
+## Describe the bug
+[ Replace this paragraph with a clear and concise description of what the bug is. ] 
+
+## Describe how to reproduce the bug
+[ Replace this with steps to reproduce the behavior:
 
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
-4. See error
+4. See error ]
 
-**Kirby version**
+## Which Kirby version was used?
 - [e.g. 1.1.10]
 
-**Expected behavior**
-<!--A clear and concise description of what you expected to happen...-->
+## What was the expected behavior?
+[ Replace this paragraph with a clear and concise description of what you expected to happen ]
 
-**Screenshots**
-<!--If applicable, add screenshots to help explain your problem.-->
+## Add any screenshots
+[ Replace this paragraph with screenshots to help explain your problem (if applicable) ]
 
-**Please complete the following information:**
+## Please complete the following information:
 - OS: [e.g. Windows, MacOS, iOS]
 - Browser [e.g. Chrome, Safari]
 - Browser version [e.g. 22]
 
-**Additional context**
-Add any other context about the problem here.
+## Are there any additional context?
+[ Replace this paragraph with any additional context about the problem here (if any) ]
+
+<hr />
+
+## Checklist:
+
+The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.
+
+### Verification
+To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 
 
-<!--**Process around fixing and prioritization**-->
-<!--In order to fix bugs in a timely fashion please follow these few steps when relevant-->
+- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](../CONTRIBUTING.md/#team-kirby); do not proceed until this is done
```

Comment:
`[team Kirby]` should lead to `../SUPPORT.md/#team-kirby`, no?

Reply:
Yup - solved